committing.rst

```diff
@@ -204,9 +204,28 @@ column width should be used. There is no indentation or leading marker in the
 file (e.g. ``-``). There is also no need to start the entry with the issue
 number as it's part of the file name itself. Example news entry::
 
-  Fix warning message when ``os.chdir()`` fails inside
-  ``test.support.temp_cwd()``.  Patch by Chris Jerdonek.
+   Fix warning message when ``os.chdir()`` fails inside
+   ``test.support.temp_cwd()``.  Patch by Chris Jerdonek.
 
+In addition to inline reST, Misc/NEWS entries also support the usage of
+`Sphinx roles <https://devguide.python.org/documenting/#id4>`_. Several
+commonly used roles include :func:, :class:, and :meth:. When a
+corresponding entry is found within the documentation, the text within
+the role is converted into a hyperlink. Example news entry with Sphinx::
+
+   Fix warning message when :func:`os.chdir` fails inside
+   :func:`test.support.temp_cwd`. Patch by Chris Jerdonek.
+
+The inline Sphinx roles should be used to assist readers in finding more
```

Comment:
This "should be used to assist" is unclear.  It seems this paragraph strongly recommends to use the roles.
Maybe, "should be used only to assist" or "can be used"?

Reply:
Agreed, I think the current wording may come off a bit too strongly here. I'll change it to "can be used".